Earthfile

```
@@ -638,7 +638,7 @@ all-buildkitd:
         ./buildkitd+buildkitd --BUILDKIT_PROJECT="$BUILDKIT_PROJECT"
 
 dind-alpine:
-    BUILD +dind --OS_IMAGE=alpine --OS_VERSION=3.18 --DOCKER_VERSION=23.0.6-r5
+    BUILD +dind --OS_IMAGE=alpine --OS_VERSION=3.18 --DOCKER_VERSION=23.0.6-r6
```

alpine packages are a pain, we could use ranges, i.e.
```suggestion
    BUILD +dind --OS_IMAGE=alpine --OS_VERSION=3.18 --DOCKER_VERSION=~23.0.6-r6
```
but maybe that would go against the explicit image tag.

Alternatively I'll add Renovate as part of my work to try to catch those upgrades faster before packages disappear.

---

I tried to use `23.0.6` (without the release candidate) but it led to an error. I presume that versions without RC will stick around longer. :shrug:

---

the tilda _might_ make it work.

---

The `+dind` target seems to fail with the tilde. I'll take another look later.

---

no worries, maybe the tilda needs to be added in the script because the arg is used for the image tag.
But anyway I'm working on the script, UDC and everything related so I'll take a look.